src/main/java/ie/gmit/Prescription.java

```diff
@@ -0,0 +1,46 @@
+package ie.gmit;
+
+public class Prescription extends Item {
+    private String prescriptionID;
+    private boolean prescription;
+
+    public Prescription(String name, double price, int quantity, String prescriptionID, boolean prescription) {
+        super(name, price, quantity);
+        setPrescriptionID(prescriptionID);
+        setPrescription(prescription);
```

Comment:
Should we also have a prescription length or amount? Like they can get the prescription 3 times total?

Reply:
Yea I think that would be a good idea to get some content and functionality